autogpts/autogpt/autogpt/config/config.py

```diff
@@ -134,6 +134,11 @@ class Config(SystemSettings, arbitrary_types_allowed=True):
         from_env=lambda: _safe_split(os.getenv("DISABLED_COMMAND_CATEGORIES")),
     )
 
+    disabled_commands: list[str] = UserConfigurable(
+        default_factory=list,
+        from_env=lambda: _safe_split(os.getenv("DISABLED_COMMANDS")),
+    )
+
```

Comment:
Add to `.env.template`

Reply:
Done [Update .env.template and docs](https://github.com/Significant-Gravitas/AutoGPT/pull/7054/commits/9f674c27c8ec1a8daae709f6a4a4766e7f66f1e1)